src/inspector/node_protocol.pdl

```diff
@@ -37,3 +37,58 @@ experimental domain NodeTracing
   # Signals that tracing is stopped and there is no trace buffers pending flush, all data were
   # delivered via dataCollected events.
   event tracingComplete
+
+# Supports additional targets discovery and allows to attach to them.
```

Comment:
This description should probably talk about workers as well.

Reply:
Done.